app/src/main/java/com/poupa/vinylmusicplayer/ui/activities/tageditor/AbsTagEditorActivity.java

```diff
@@ -386,9 +386,7 @@ void writeValuesToFiles(@NonNull final Map<FieldKey, String> fieldKeyValueMap, @
                 writeTagsApi21_SAFGuide.launch(new Intent(this, SAFGuideActivity.class));
             }
         } else {
-            if (SAFUtil.isSDCardAccessGranted(this)) {
-                writeTags(savedSongs);
-            }
+            writeTags(savedSongs);
```

Comment:
I had to do this in passing, otherwise nothing would happen on save for Android 33.
It's probably safe, because the activity asks for permission to modify the audio files at startup?

Reply:
Hmm, this is a bug. If the `isSDCardAccessGranted` returns false, the app should bring up the permission request popup (as in the case of tag editor).

If you trigger the tag editor, you should get the function to return true as expected.

I'd rather restore this check for now.

Reply:
Even on API 30+?

For me (android 13, on my phone, and in the emulator):
* A permission dialog ("Allow Vinyl Media Player to modify this audio file?") is shown when the edit dialog is *open*
* isSDCardAccessGranted always returns false without showing a permission dialog
* writeTags works fine (and I think this is because of the permission dialog shown when the dialog opens)

This is not strictly related to this PR, so happy to revert this change and open another issue about what I'm seeing on android 13 if you prefer.

Reply:
Yes, please create another PR

Reply:
I've reverted the change and created #863.